pages/tutorials/connecting_kubernetes.md.erb

```diff
@@ -0,0 +1,167 @@
+# Connecting to Kubernetes
+
+Your BuildKite agents can connect to any Kubernetes cluster. That
+depends on where your agents and Kuberneters cluster run. This
+tutorial covers common deployment scenarios for AWS, EKS, GCP, GKE,
+and self provisioned clusters.
+
+<%= toc %>
+
+## AWS & EKS
+
+<%= estimated_time "15-20 minutes" %>
+
+These demonstrate how to grant Buildkite Agents `kubectl` access to an
+EKS cluster using AWS IAM roles.
+
+The steps assume your agents are deployed to EC2 with a dedicated
+instance profile and associate IAM role. If you're using the [Elastic
+Stack][] then you're set. Second, it assumes the agents have access to
+the Kubenetes API admin.
+
+<div class="Docs__troubleshooting-note">
+<p>
+If your EKS cluster's Admin API access is set to private, then
+you'll need to update networking to allow communication between the agents and
+the admin API before continuing. However this outside the scope of this
+tutorial.
+</p>
+</div>
+
+### Configuring the IAM Role
+
+You'll need to grant EKS access to the IAM role associated with
+your agents. Attaching an existing AWS Managed Policy is the easiest
+way. The `PowerUser` policy is easiest for this tutorial since it
+grant access to all AWS APIs execpt IAM management.
+
+If you're using the ElasticStack, then open the CloudFormation console and set
+the `ManagedPolicyARN` parameter to `arn:aws:iam::aws:policy/PowerUserAccess`.
```

Comment:
Howe can I prevent emojification of these ARNs? They're being replaced with `<img>` tags inside a code tag.

Reply:
If you escape the colons, it should sort itself out! I just tested out this: `arn\:aws\:iam:\:aws\:policy/PowerUserAccess` and it came up emoji-free.